app/Repositories/Dislike/DislikeRepository.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Hedonist\Repositories\Dislike;
+
+use Hedonist\Entities\Dislike\Dislike;
+use Hedonist\Entities\Place\Place;
+use Prettus\Repository\Eloquent\BaseRepository;
+use Prettus\Repository\Criteria\RequestCriteria;
+use Prettus\Repository\Contracts\CriteriaInterface;
+use Illuminate\Database\Eloquent\Collection;
+
+class DislikeRepository extends BaseRepository implements DislikeRepositoryInterface
+{
+    public function model()
+    {
+        return Dislike::class;
+    }
+
+    public function save(Dislike $dislike): Dislike
+    {
+        $dislike->save();
+
+        return $dislike;
+    }
+
+    public function getById(int $id): ?Dislike
+    {
+        return Dislike::find($id);
+    }
+
+    public function findAll(): Collection
```

Comment:
We unlikely will use such method in our app to query dislike. Please delete for now.